dasharo-security/tpm2-commands.robot

```diff
@@ -312,6 +313,43 @@ TPMCMD014.001 Encrypt and Decrypt non-rootfs partition (Ubuntu 22.04)
     Execute Command In Terminal    rm -f key seal.* prim.* test-partition
     Execute Linux Tpm2 Tools Command    tpm2_evictcontrol -c 0x81010001
 
+TPMCMD015.001 Encrypt and Decrypt rootfs partition (Ubuntu 22.04)
+    [Documentation]    Test encrypting and decrypting rootfs partition using
+    ...    TPM. This test assumes that Ubuntu already has encrypted rootfs. This
+    ...    test is executed if rootfs_encryption_test variable is True (can be
```

Comment:
@DaniilKl this needs an update.

Reply:
Updated here 2dd9c61d16532b6f907307ce2293dde54dffec91